doc/classes/EditorInspector.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="4.0">
 	<brief_description>
+		A tab used to edit the properties of the selected node.
 	</brief_description>
 	<description>
+		Editor inspector is by default located on the right-hand side of the editor. Its used to edit the properties of the selected node. For example, you can select a node such as the Sprite2D then edit it's transform through the inspector tool. Editor inspector is an essential tool in the game development workflow.
```

Comment:
You changed the wrong its. You need to make what is currently "its" into "it's" and what is currently "it's" into "its".

"Editor inspector" should have "The" in front of it in both instances.